task-sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -158,6 +183,9 @@ def get_template_context(self) -> Context:
 
         validated_params = process_params(self.task.dag, self.task, dag_run_conf, suppress_exception=False)
 
+        self.log_url = self.get_log_url_from_ti()
+        self.mark_success_url = self.get_mark_success_url()
```

Comment:
Can we move this logic to "startup" instead? It doens't necessarily need to have context in order to set these.